src/open_inwoner/openzaak/clients.py

```diff
@@ -40,11 +40,32 @@
 
 
 class ZakenClient(APIClient):
+    def fetch_cases(
+        self,
+        user_bsn=None,
+        user_kvk_or_rsin=None,
+        max_requests=4,
+        identificatie=None,
+        vestigingsnummer=None,
+    ):
+        if user_bsn:
```

Comment:
For correctness we could check if not both `user_bsn` and `user_kvk_or_rsin` are supplied as they're mutually exclusive. Or `vestigingsnummer` and `user_bsn` etc.

Also some typing in the signature would be good.